backend/sprint/api_project/tests.py

```diff
@@ -66,8 +66,8 @@ def test_m_project(self):
 
     def test_project_detail(self):
         client = Client()
-        url = self.url+'detail/1/'
-        url2 = self.url+'detail/10/'
+        url = self.url+'detail/'+str(self.project1.pk)+'/'
```

Comment:
looks good to me to use pk rather than id. let's apply this to the other files, too.